extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -169,6 +174,66 @@ export class WizardController {
 			}));
 	}
 
+	private async createLoginWizard(stateModel: MigrationStateModel): Promise<void> {
```

Comment:
would it make sense to create a class that derives from this.  and adds specific implementations.  Adding here might be overloading it.

Reply:
added a new follow up task to handle this: https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2133709